### DISCUSSION TITLE
How to make website appear on google search

### DISCUSSION THREAD

**Discussion by SakaizaRajery** [greenelab/lab-website-template#149] [General] [answered]
Hello, 

I am currently using the website template from the Greene Lab to Update a website. 

https://kechrisk.github.io/KechrisLabWebsite/

I am trying to make the website appear on google search, but I cannot seem to figure it out. I followed the prompt through Google Console, but it seems that it expects the index file to be an html, I got stuck on step 4. 

Below is the prompt I followed:

https://stackoverflow.com/questions/26199705/github-repository-not-listing-in-google-search

Thank you!

**Comment by vincerubinetti** [comment, marked as answer]
Did you use `https://kechrisk.github.io/KechrisLabWebsite/index.html` as the url you submitted? According to that StackOverflow answer, it seems like Google wants to see the full path to the HTML file, e.g. with the `/index.html`.

What exact warning message did you get? Screenshots would help a lot. In general, for me to help you, please include more detail; as much as you possibly can.

**Reply by SakaizaRajery** [reply]
Thanks a lot for the response. I actually used https://kechrisk.github.io/KechrisLabWebsite/index.md , the error, says something like: file not found. Not sure if i tried changing it to html. I will try that and see how it goes. I will add screenshots if it still doesn't work.

**Reply by vincerubinetti** [reply]
`index.md` will not be a part of your live website, it will only be in your source code. Web browsers can't natively understand or display markdown files; Jekyll/the template converts markdown to `.html` files when your site is built and before it is deployed to live.

Either using `https://kechrisk.github.io/KechrisLabWebsite` or `https://kechrisk.github.io/KechrisLabWebsite/index.html` should work.
